src/settings-ui/Settings.UI/ViewModels/MouseUtilsViewModel.cs

```diff
@@ -48,7 +48,7 @@ public MouseUtilsViewModel(ISettingsUtils settingsUtils, ISettingsRepository<Gen
             }
 
             FindMyMouseSettingsConfig = findMyMouseSettingsRepository.SettingsConfig;
-            _findMyMouseActivationMethod = FindMyMouseSettingsConfig.Properties.ActivationMethod.Value;
+            _findMyMouseActivationMethod = FindMyMouseSettingsConfig.Properties.ActivationMethod.Value < 2 ? FindMyMouseSettingsConfig.Properties.ActivationMethod.Value : 0;
```

Comment:
This shouldn't be here?

Reply:
A bit outside from the PR scope but needed to handle the invalid settings value, especially since we are going to add new activation method in https://github.com/microsoft/PowerToys/pull/28096

While other invalid values (example negative ones) are handled correctly by the XAML controls, `FindMyMouseActivationMethod` is binded to the `ComboBox` `SelectedIndex` resulting in a crash.

https://github.com/microsoft/PowerToys/blob/efee03eb99732a28e7e861e612abd2cf4984fada/src/settings-ui/Settings.UI/SettingsXAML/Views/MouseUtilsPage.xaml#L41-L46

Reply:
Got it. Thanks